### ISSUE TITLE
bug: Missing extension Int64: JSValue {} in JSTypes iOS

### ISSUE THREAD

**Issue by jepiqueau** [ionic-team/capacitor#5538]
### Capacitor Version

```
Latest Dependencies:

  @capacitor/cli: 3.4.3
  @capacitor/core: 3.4.3
  @capacitor/android: 3.4.3
  @capacitor/ios: 3.4.3

Installed Dependencies:

  @capacitor/cli: 3.4.2
  @capacitor/core: 3.4.2
  @capacitor/android: 3.4.2
  @capacitor/ios: 3.4.2
```

### Current Behavior

Does not accept Int64 in JSObject return {} in JSObject

### Other Technical Details

`npm --version` output:
npm: '8.1.2'
`node --version` output:
node: '16.13.2'
`pod --version` output (iOS issues only):
1.10.0

**Comment by jcesarmobile** [comment]
Can you provide a plugin project or sample code where you return the Int64?

I have no problems returning Int64 like this:
```
let myInt64Value: Int64 = 12345678901
call.resolve([
    "myInt64Value": myInt64Value
])
```

**Comment by jepiqueau** [comment]
@jcesarmobile  Sorry as my code was not working i completely delete it.
i was doing something like this
```
var result: JSObject =  JSObject()
result["age"]  = the returned int64 from the sql query   as JSValue
result["message] = the returned text from the sql query   as JSValue
call.resolve(["values": result])
```
and for this to work i had to add extension Int64: JSValue {} in the JSTypes iOS
It was on the result of a sqlite query which return a [[String: Any]] and i was trying to order the dictionary keys without success see issue#195 (Query returning multiple rows) of the @capacitor-community/sqlite. This seems to be a problem for them as this is not equivalent to the result of Android or Electron where it is real JSObject which are returned and so the keys in the order they have been created  which is not the case for iOS.

so my answer to the issue#195
```
After so many trials, nothing work.

In fact reading the Capacitor documentation for plugin iOS Returning Data Back

The resolve() method of CAPPluginCall takes a dictionary and supports JSON-serializable data types.
and according to the Apple doc

A Dictionary is a generic type that manages an unordered collection of key-value pairs.
So Basically you have to deal with this in your app by recreating a json object from the return json object.

In Android and javascript you deal with json object directly
```
Voilà, if you have an idea on how to deal with this. In a mean time i will close this issue even if i think it is not worth to add the extension Int64
